pkgs/servers/sql/materialize/default.nix

```diff
@@ -40,17 +40,17 @@ let
 in
 rustPlatform.buildRustPackage rec {
   pname = "materialize";
-  version = "0.8.0";
-  rev = "b2fe225f1afcfec4912976bdaa4a44caf3ca0842";
+  version = "0.8.1";
+  rev = "ef996c54db7c9504690b9f230a4a676ae1fb617f";
 
   src = fetchFromGitHub {
     owner = "MaterializeInc";
     repo = pname;
     inherit rev;
```

Comment:
```suggestion
    rev = "v${version}"
```

Reply:
I'd like to leave it as is because the commit hash is compiled into the binary (see the `MZ_DEV_BUILD_SHA` attribute further down) and this ensures they are in sync.

Reply:
The tag needs to point to the commit sha. If you are fetching the rev this needs to be verified on every update. Fetching the tag does not because github does not share them between forks. It makes no difference when building but makes updates for reviewers easier.

Reply:
That makes sense, fixed